src/tests/end-to-end/common/element-identifiers/common-selectors.ts

```diff
@@ -11,3 +11,9 @@ export const CommonSelectors = {
 export const GuidanceContentSelectors = {
     mainContentContainer: '.content-container',
 };
+
+export const fastPassSelectors = {
+    tabstopNavButtonSelector: 'a[title="Tab stops"]',
```

Comment:
should it be "tab**S**topNavButtonSelector"? to make it match the other names case styles?

Reply:
Will update.